plugin/vimrtags.py

```diff
@@ -33,10 +33,13 @@ def run_rc_command(arguments, content = None):
             input = content,
             stdout = subprocess.PIPE,
             stderr = subprocess.PIPE,
```

Comment:
By the way, shouldn't there be `subprocess.STDOUT`, as in other branches? 3.5 doesn't change behaviour, from what I've checked.